package.json

```diff
@@ -7,11 +7,14 @@
     "build:sassdoc": "sassdoc --parse node_modules/govuk-frontend/govuk/ > data/sassdoc.json",
     "scripts": {
       "lint": "standard"
-    }
+    },
+    "test": "hyperlink -ri deploy/public/index.html | tap-spot"
```

Comment:
Just a suggestion but could this be
```
"test": "bundle exec middleman build && hyperlink -ri deploy/public/index.html | tap-spot"
```
so that you don't need to run the build separately to see any errors? Travis could also then just run `npm test`.